src/command.c

```diff
@@ -70,6 +70,9 @@ static bool cmd_target_power(target_s *target, int argc, const char **argv);
 #ifdef PLATFORM_HAS_BATTERY
 static bool cmd_target_battery(target_s *t, int argc, const char **argv);
 #endif
+#ifdef PLATFORM_HAS_WIFI
+static bool cmd_wifi(target_s *t, int argc, const char **argv);
+#endif
```

Comment:
Not for this PR, but for a future one: These should probably both become platform commands. That way you don't need to export as much platform state into the rest of the code base, and they get grouped under the platform commands heading in the `mon help` output. We can look at improving this in v2.1 though so this is fine as is for now.

Reply:
Agreed as previously discussed.